packages/runtime/src/runtime.ts

```diff
@@ -66,6 +69,38 @@ export class Runtime {
     this.timestamp = 1;
   }
 
+  get defaultBalance (): number {
+    return 1e9; // 1000 Algos
+  }
+
+  /**
+   * Returns a list of initialized default accounts created using static accountSDK from account.ts
+   *  and funded with default balance
+   * @returns list of AccountStore
+   */
+  _setupDefaultAccounts (): AccountStore[] {
+    const balance = this.defaultBalance;
+    const accounts = Object.values(defaultSDKAccounts)
+      .map(accountInfo => new AccountStore(balance, accountInfo));
+    this.initializeAccounts(accounts);
+    return accounts;
+  }
+
+  /**
+   * Resets the state of the default accounts
+   */
+  resetDefaultAccounts (): void {
+    this._defaultAccounts = this._setupDefaultAccounts();
+  }
+
+  /**
+   * Getter for _defaultAccounts, returns a synced version of the accounts list
+   * @returns list of AccountStore
+   */
+  defaultAccounts (): AccountStore[] {
+    return this._defaultAccounts.map((account) => this.getAccount(account.address));
```

Comment:
hmm, could you please explain this? We're already inside runtime, do we need to use "this.getAccount" (sync) here? Sync is required when you declare variables outside the runtime's context (eg. in a test), but need to use the updated state after each transaction.

Reply:
I'm pretty sure, that after transaction, runtime will create a new instance (copy) of account, so the `this._defaultAccounts` won't be updated.